core/api/src/services/svix/index.ts

```diff
@@ -31,15 +31,36 @@ function prefixObjectKeys(
   }, {})
 }
 
-export const CallbackService = (config: SvixConfig) => {
+interface CallbackService {
```

Comment:
please move to https://github.com/GaloyMoney/galoy/blob/main/core/api/src/domain/callback/index.types.d.ts

Reply:
also please rename to `ICallbackService`